cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -20,17 +20,16 @@ import (
 	"io"
 	"os"
 
+	"github.com/mitchellh/mapstructure"
```

Comment:
Revert this

Reply:
hun .. should be == master .. good catcher .. missed up in the squash.